style.css

```diff
@@ -0,0 +1,89 @@
+/* style sheet for rocket science home */
+
+body {
+        background-image:url("background.png");
+        background-repeat:repeat-y;
+        max-width: 2049px;
+     }
+
+header {
+          text-align:center;
+          border-bottom-style:solid;
+          border-bottom-color:white;
+          border-bottom-width:1px
+       }
+
+header p {
+            width:650px; 
+            display:inline-block;
+            font-family:"Gotham Book", sans-serif;
+            font-size:14pt;
+            color:#FFF;
+         }
+
+section {}
+
+h1 {
+      font-family:"Gotham", sans-serif;
+      font-size:16pt;
+      font-weight:bold;
+      color:#FFF;
+   }
+
+h2 {
+      font-family:"Gotham Book", sans-serif;
+      font-size:14pt;
+      color:#FFF;
```

Comment:
Color hex codes should always be written lower-case. The difference is almost zero, but there is one tiny reason to choose lowercase over uppercase, and consistency is important. Lowercase is slightly easier to read at a glance!